README.md

```diff
@@ -237,469 +103,115 @@ You need to copy the following files:
-- Peter Nelson (peter1138) - Spiritual descendant from newGRF gods (since 0.4.5)
-- Ingo von Borstel (planetmaker) - General coding, Support (since 1.1)
-- Remko Bijker (Rubidium) - Lead coder and way more (since 0.4.5)
-- José Soler (Terkhen) - General coding (since 1.0)
-- Leif Linse (Zuu) - AI/Game Script (since 1.2)
-
-### Inactive Developers:
-
-- Jean-François Claeys (Belugas) - GUI, newindustries and more (0.4.5 - 1.0)
-- Bjarni Corfitzen (Bjarni) - macOS port, coder and vehicles (0.3 - 0.7)
-- Victor Fischer (Celestar) - Programming everywhere you need him to (0.3 - 0.6)
-- Jaroslav Mazanec (KUDr) - YAPG (Yet Another Pathfinder God) ;) (0.4.5 - 0.6)
-- Jonathan Coome (Maedhros) - High priest of the NewGRF Temple (0.5 - 0.6)
-- Attila Bán (MiHaMiX) - WebTranslator 1 and 2 (0.3 - 0.5)
-- Zdeněk Sojka (SmatZ) - Bug finder and fixer (0.6 - 1.3)
-- Christoph Mallon (Tron) - Programmer, code correctness police (0.3 - 0.5)
-- Patric Stout (TrueBrain) - NoProgrammer (0.3 - 1.2), sys op (active)
-- Thijs Marinussen (Yexo) - AI Framework, General (0.6 - 1.3)
-
-### Retired Developers:
-
-- Tamás Faragó (Darkvater) - Ex-Lead coder (0.3 - 0.5)
-- Dominik Scherer (dominik81) - Lead programmer, GUI expert (0.3 - 0.3)
-- Emil Djupfeld (egladil) - macOS port (0.4 - 0.6)
-- Simon Sasburg (HackyKid) - Bug fixer (0.4 - 0.4.5)
-- Ludvig Strigeus (ludde) - Original author of OpenTTD, main coder (0.1 - 0.3)
-- Cian Duffy (MYOB) - BeOS port / manual writing (0.1 - 0.3)
-- Petr Baudiš (pasky) - Many patches, newgrf support, etc. (0.3 - 0.3)
-- Benedikt Brüggemeier (skidd13) - Bug fixer and code reworker (0.6 - 0.7)
-- Serge Paquet (vurlix) - 2nd contributor after ludde (0.1 - 0.3)
-
-### Thanks to:
-
-- Josef Drexler - For his great work on TTDPatch.
-- Marcin Grzegorczyk - For his TTDPatch work and documentation of Transport Tycoon Deluxe internals and track foundations
-- Stefan Meißner (sign_de) - For his work on the console
-- Mike Ragsdale - OpenTTD installer
-- Christian Rosentreter (tokai) - MorphOS / AmigaOS port
-- Richard Kempton (RichK67) - Additional airports, initial TGP implementation
-- Alberto Demichelis - Squirrel scripting language
-- L. Peter Deutsch - MD5 implementation
-- Michael Blunck - For revolutionizing TTD with awesome graphics
-- George - Canal graphics
-- Andrew Parkhouse (andythenorth) - River graphics
-- David Dallaston (Pikka) - Tram tracks
-- All Translators - For their support to make OpenTTD a truly international game
-- Bug Reporters - Thanks for all bug reports
-- Chris Sawyer - For an amazing game!
+
+## 1.5) Add-on content / mods
+
+OpenTTD features multiple types of add-on content, which modify gameplay in different ways.
+
+Most types of add-on content can be downloaded within OpenTTD via the 'Check Online Content' button in the main menu.
+
+Add-on content can also be installed manually, but that's more complicated; the [OpenTTD wiki](https://wiki.openttd.org/OpenTTD) may offer help with that.
```

Comment:
Useless link is useless. At least should link to the NewGRF page

Reply:
Maybe link to directory structure doc

Reply:
Linking to the NewGRF page doesn't cover AIs, GS, scenarios, heightmaps....